VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls-base.component.ts

```diff
@@ -125,6 +149,12 @@ export abstract class HearingControlsBaseComponent implements OnInit, OnDestroy
     }
 
     ngOnDestroy(): void {
+        this.participantSpotlightUpdateSubscription?.unsubscribe();
+        this.participantSpotlightUpdateSubscription = null;
```

Comment:
is it necessary to set them to null?

Reply:
Feel like it is good practice, will stop unsubscribe getting called twice, plus free up memory as there should be no other reference to the subscription.